<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>COM Port</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxCOMPort">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>211</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Close COM Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>BAUD Rate</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditRx">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>512</width>
        <height>100</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>COM Port</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxBAUDRate">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>211</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>additonal settings are '8,N,1'...</string>
      </property>
      <property name="text">
       <string>Open COM Port</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditTx">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>512</width>
        <height>100</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Serial RX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Serial TX</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Node setup</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditSecurityKey">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditChannel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Security Key</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAddress">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPANID">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Configure Node</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PANID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Channel</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxChannelAck">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Channel OK</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxAddressOK">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Address OK</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxPanIdOK">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>PANID OK</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxSecurityKey">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Security Key OK</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Command Frame Constructor</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxCmd">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>281</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Command Selection:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditStartByte">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>70</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0xAB</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>50</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Byte</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSizeByte">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>70</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>103</x>
        <y>50</y>
        <width>30</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCmdId">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>70</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CMD ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCRC">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>64</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>484</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CRC</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>298</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Payload</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPayload">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>501</width>
        <height>111</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="twoByteCmd">
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Command Option:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxCmdOption">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>256</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_x">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>label_x</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="threeByteCmd">
       <widget class="QLabel" name="label3Byte">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string>This Node's Address:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditThisAddress">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Enter an address in the form '12CD'.</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="fourByteCmd">
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string>Sleep Interval (milliseconds):</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSleep">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Enter the number of miliseconds to sleep.</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="UART">
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>62</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Data Bits</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxDataBits">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>80</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>0</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxParity">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>80</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>0</y>
          <width>62</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop Bits</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxStopBits">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>0</y>
          <width>62</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxBaudRate">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>20</y>
          <width>121</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Data">
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Destination Address</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditDestAddr">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>0</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Options</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxOptions">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>17</y>
          <width>161</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>0</y>
          <width>62</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Handle</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditHandle">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Payload</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditUserPayload">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonClear">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>80</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>311</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Serial fame to send:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditFrameToSend">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>511</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTxFrame">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>280</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Frame</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioBlock Response:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditRfResponse">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>511</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Sniffer</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetSniffer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>681</width>
        <height>561</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>12</number>
      </property>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>640</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
